src/linter/root.go

```diff
@@ -1456,6 +1464,20 @@ func (d *rootWalker) parseFuncArgs(params []ir.Node, parTypes phpDocParamsMap, s
 	return args, minArgs
 }
 
+func (d *rootWalker) typeHintHasMoreAccurateType(typeHintType, phpDocType types.Map) bool {
+	// if is not array typehint
```

Comment:
We usually write comments differently in noverify.
With periods and sentences start with a capital letter.

Reply:
Done.